scripts/android/register-app-version.sh

```diff
@@ -35,7 +33,7 @@ fi
 # Get values based on environment
 case ${APP_ENVIRONMENT} in
   staging)
-    token_url="https://partner-abt.staging.entur.org/oauth/token"
+    token_url="https://partner.staging.entur.org/oauth/token"
```

Comment:
This should just be fetched from the secret, it's provided in the `tokenUrl` field.